NAS-102733 minor update to docs about exporting pool (#1362)
freenas/freenas-docs: userguide/storage.rst

Export/Disconnect a Pool
~~~~~~~~~~~~~~~~~~~~~~~~

> :guilabel:`Export/Dissonnect` removes a pool from the %brand%

Reviewer: Typo.

> system, making the disks in the pool available. The data in the pool

Reviewer: It mainly disconnects a pool.  The disks are only made available if they are erased, but that should not be the default.
"Export/Disconnect is used to cleanly disconnect a pool from the system. This is used before physically disconnecting the pool so it can be imported on another system, or to optionally detach and erase the pool so the disks can be reused."

   | Setting                           | Description                         |
   |                                   |                                     |
   +===================================+=====================================+

>    | Destroy data on this pool?        | Destroys all data on the disks in   |

Reviewer: The table is a nice idea!
s/Destroys/Destroy/  (present tense)

   |                                   | the pool. **This action cannot be   |
   |                                   | undone**.                           |
   |                                   |                                     |
   +-----------------------------------+-------------------------------------+
   | Delete configuration of shares    | Delete any share configurations     |
   |                                   | set up on the pool.                 |
   |                                   |                                     |
   +-----------------------------------+-------------------------------------+
   | Confirm export/disconnect         | Confirm the export/disconnect       |
   |                                   | operation.                          |
   |                                   |                                     |
   +-----------------------------------+-------------------------------------+

> An encrypted pool also displays :guilabel:`DOWNLOAD KEY` to download

Reviewer: ```suggestion
If the pool is encrypted, :guilabel:`DOWNLOAD KEY` is also shown to download
```

the :ref:`encryption key <Encryption and Recovery Keys>` for that pool.

To :guilabel:`Export/Disconnect` the pool and keep the data and
configurations of shares, set **only**
:guilabel:`Confirm export/disconnect` and click

> :guilabel:`EXPORT/DISCONNECT`. Do this when disconnecting the pool.

Reviewer: "Do this" was not meant to be literal.
```suggestion
:guilabel:`EXPORT/DISCONNECT`.
```

> This makes it possible to re-import the pool at a later time.

Reviewer: Moved up.
```suggestion
```
